include/spatio_temporal_voxel_layer/spatio_temporal_voxel_layer.hpp

```diff
@@ -70,6 +70,7 @@
 #include <tf/message_filter.h>
 #include <tf/exceptions.h>
 
+#include <boost/thread/mutex.hpp>
```

Comment:
Make a "boost" section after STL section of header imports

Reply:
Oh, I actually meant to remove this since it's no longer being used